### DISCUSSION TITLE
client request invalid:429

### DISCUSSION THREAD

**Discussion by chacha121** [Shabinder/SpotiFlyer#1825] [Q&A] [answered]
When I paste the Spotify link on the android app it  shows error
**_Client request invalid: 429. Text :"too many requests"_**
and I've been trying it for at least 6 hours......

**Comment by mathi300** [comment]
Same problem Spotify song not download all

Please help

**Comment by mathi300** [comment]
> When I paste the Spotify link on the android app it shows error **_Client request invalid: 429. Text :"too many requests"_** and I've been trying it for at least 6 hours......

Same problem Spotify song  not download

**Comment by T-vK** [comment, marked as answer]
See #1814
